src/data/Checkout.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react'
+import formatPrice from '../helpers/formatPrice'
+import './Checkout.css'
+
+export default class Checkout extends Component {
+    constructor() {
+        super()
+        this.state = {
+            firstName:'',
+            lastName:'',
+            email: '',
+            creditCard: '',
+            zipCode: ''
+        }
+    }
+
+    submit = (e) => {
+        e.preventDefault()
+        const { firstName,lastName,email,creditCard,zipCode } = this.state
+
+        if (!firstName || !lastName || !email || !creditCard || !zipCode) {
+            window.alert("Input is not valid")
+        }else if (creditCard.length !== 16) {
+            window.alert('Credit card number is not valid')
+        } else if (zipCode.length !== 5) {
+            window.alert('Zip code is not valid') 
+        } else {
+            window.alert('Purchase complete. You will be charged' + '${formatPrice(price)}' )
+        }
+
+    }
+
+    handleChange = (e) => {
+        this.setState({ [e.target.id]: e.target.value})
```

Comment:
So you could use .id here like you are, but I think the better way is to use `.name`. You'd just have to add a `name=""` attribute to every input below.